### DISCUSSION TITLE
Increasing Font Size - Line Height

### DISCUSSION THREAD

**Discussion by frankwb24** [blinksh/blink#1862] [Support] [answered]
Hi,
I'm trying to increase the font size in Blink but when I increase it either in the configuration or using the pinch gesture only the width of the font size changes the line height does not. I have tried multiple fonts but all behave the same. It's not readable with the line size remaining constant. Is there a way I can increase the line height to match the width?
This is on iPadOS 16. 
Thanks.

**Comment by carloscabanero** [comment]
Hi! That's a really weird issue. What version of Blink are you running? Did you compile it yourself? Could you send us a screenshot so we can see what is going on? Have you tried to set the font size also as part of config > Appearance? Does this also happens with the default font?

Thanks!

**Reply by frankwb24** [reply, marked as answer]
It was happening either with pinch or using config > Appearance. I was able to fix it by uninstalling and reinstalling Blink. Thanks!

**Reply by carloscabanero** [reply]
Glad you figured it out. That sounded like a weird issue with the OS itself.
